docs/_Index.md

```diff
@@ -0,0 +1,16 @@
+---
+github: https://github.com/JoinColony/colonyNetwork
+logo: colonyNetwork.svg
+section: index
+sectionOrder:
+  - Docs
+  - API
+---
+
+![logo](https://raw.githubusercontent.com/JoinColony/colonyNetwork/develop/docs/img/colonyNetwork_combomark.svg?sanitize=true)
+
+**A platform for open organizations.**
+
+The contracts that comprise the Colony Network provide a standard framework for the creation of decentralized organizations as described in the [Colony Whitepaper](https://colony.io/whitepaper.pdf).
+
+[Read colonyNetwork docs](/colonynetwork/docs-colony/ "Docs")
```

Comment:
This is a comment for the file below (`Colony_Header.png`). Please remove that and use the colonyNetwork comboMark svg instead. Please also adjust the path here: https://raw.githubusercontent.com/JoinColony/colonyNetwork/develop/docs/README.md (first line)